tests/e2e/framework/multicluster.go

```diff
@@ -84,30 +84,36 @@ func (k *KubeInfo) generateRemoteIstio(src, dst string) error {
 		return err
 	}
 	getOpt := meta_v1.GetOptions{IncludeUninitialized: true}
-	var pilotEPS, mixerEPS *v1.Endpoints
+	var pilotEPS, policyEPS, statsdEPS *v1.Endpoints
 
 	for i := 0; i <= 200; i++ {
 		pilotEPS, err = k.KubeClient.CoreV1().Endpoints(k.Namespace).Get("istio-pilot", getOpt)
 		log.Infof("PilotEPS = %s ", pilotEPS)
 		if (len(pilotEPS.Subsets) != 0) && (err == nil) {
 
-			mixerEPS, err = k.KubeClient.CoreV1().Endpoints(k.Namespace).Get("istio-policy", getOpt)
-			log.Infof("mixerEPS = %s ", mixerEPS)
-			if (len(mixerEPS.Subsets) != 0) && (err == nil) {
+			policyEPS, err = k.KubeClient.CoreV1().Endpoints(k.Namespace).Get("istio-policy", getOpt)
+			log.Infof("policyEPS = %s ", policyEPS)
+			if (len(policyEPS.Subsets) != 0) && (err == nil) {
+				break
```

Comment:
This logic is wrong.  You can't break here as you will skip setting up the statsd EP